lib/pbench/server/database/alembic/versions/62eddcec4817_add_dataset_access.py

```diff
@@ -0,0 +1,34 @@
+"""Add Dataset.access
+
+Revision ID: 62eddcec4817
+Revises: base
+Create Date: 2021-05-27 16:22:13.714761
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "62eddcec4817"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """
+    Add the "access" column, and set all existing rows to "private". This can't
+    be done in a single step, apparently. Instead, we add the column, set the
+    value in all existing rows, and then mark it non-nullable.
+    """
+    op.add_column("datasets", sa.Column("access", sa.String(255), default="private"))
+    op.execute("UPDATE datasets SET access = 'private'")
+    op.alter_column("datasets", "access", nullable=False)
```

Comment:
NOTE: Need to either modify this to change the type of `dataset_metadata` column `value` to `text` or create a second update file with that.

Reply:
Done